spec/commands/profiles_update_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Profiles::Update do
+  subject(:command) { described_class.new(form:) }
+
+  let(:profile) { create(:profile) }
+  let(:user) { profile.user }
+  let(:form) { ProfileForm.new(profile:, user:) }
+  let(:uploaded_resume) do
+    ActiveStorage::Blob.create_and_upload!(
```

Comment:
Use this b/c flow expects a direct upload rather than attaching the file via io.